Pipfile

```diff
@@ -31,6 +31,7 @@ pytest-flask = "~=0.10"
 pytest = "~=3.0"
 pytest-cov = "~=2.7.1"
 coveralls = "~=2.0"
+lovely-pytest-docker = "~=0.2.0"
```

Comment:
this could cause trouble w/ the changes in #461 since it triggered a change in `Pipfile.lock` which was updated in #461  too. one way forward could be to check out this branch, get rid of this line, override `Pipfile.lock` with the one in `master`, then add the above line back in, regenerate `Pipfile.lock`, then commit?

Reply:
i rebased it on top of master, so this should be fine

Reply:
apparently not:

https://github.com/orchestracities/ngsi-timeseries-api/compare/fix-444...fix-446

ouch! :-)